tests/testing-in-node.md

```diff
@@ -93,29 +109,118 @@ Time:        0.246 s, estimated 1 s
 
-We can fix this by installing the [Jest Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest). This will allow us to see the tests using the Flask icon in the editor.
+Let's fix this by installing the [Jest Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest). This will allow us to see the tests by clicking the flask icon in the editor, which switches us to the Testing panel.
 
-After installing the extension, we should now see that the Jest extension detected our tests when we click on the flask icon. 
+After installing the extension, we should now see the flask icon, indicating that the Jest extension has detected our tests. Click the flask icon to switch to the Testing panel.
 
 ![Flask icon displays Fizzbuzz tests](../assets/tests__testing-in-node__tests-detected-by-jest-extension.png)
 
-Just like with Python we can now use VS Code to run our tests and even use breakpoints to inspect and debug the tests.
+<!-- available callout types: info, success, warning, danger, secondary, star  -->
+### !callout-warning
+
+## Troubleshooting Tests in VS Code
+
+It's possible our tests won't appear even after installing the Jest extension. And sometimes tests that had previously been showing in VS Code will no longer appear.
+
+<br/>
+
+<details>
+<summary>Expand this section for some troubleshooting tips.</summary>
+
+We should first ensure there are no syntax errors in our code, either in the tests or in the code being used by the tests. Check the Problems tab in VS Code to see if there are any errors. If there are, fix them and try again. If nothing is jumping out at us, we can try running the tests from the terminal with `npm test` to see if there are any syntax errors reported there.
+
+<br/>
+
+If the tests are running from the terminal, but VS Code still doesn't see them, we may need to start the "test runner" manually. Open the Command Palette (⇧⌘P) and type `runner`. Look for the option `Jest: Start All Runners` and select it. This should start the runner and display the tests.
+
+<br/>
+
+If none of the above tips help, restarting VS Code sometimes fixes the problem (be sure to quit _all_ running instances with ⌘Q, not just the current window), as can reinstalling the Jest extension.
+
+</details>
+
+### !end-callout
+
+Just like with Python, we can now use VS Code to run our tests, and even use breakpoints to inspect and debug them.
+
+<!-- available callout types: info, success, warning, danger, secondary, star  -->
+### !callout-warning
+
+## Debug Tests from the Testing Panel, Not the Debug Panel
+
+Remember to use the Debug button that gets displayed when hovering over a test name on the Testing panel to run a test with the debugger. Alternatively, we can right-click the test status icon beside the test definition in the test code file and pick Debug Test.
+
+<br/>
+
+Do not use the Debug button on the Debug panel.
```

Comment:
Could possibly add a screenshot annotated to show the debug button (maybe w/ an X through it) and the flask button (with a circle around it), up to you.

Reply:
I know. While I was writing it, it was really asking for some screen shots and I convinced myself that we didn't need them since this isn't the first time they've encountered this. I like your suggestion of focusing on which panel they should be in rather than reproducing the equivalent set of screenshots we did for Python. That seems like a good middle ground.